pages.id/common/split.md

```diff
@@ -0,0 +1,20 @@
+# split
+
+> Memisahkan sebuah file menjadi beberapa bagian.
+> Informasi lebih lanjut: <https://www.gnu.org/software/coreutils/split>.
+
+- Memisahkan sebuah file, tiap bagian memiliki 10 baris (kecuali di bagian terakhir):
+
+`split -l {{10}} {{nama_file}}`
```

Comment:
```suggestion
`split -l {{10}} {{nama file}}`
```
I checked this with Google translate and other Indonesian translations can you update the `_` with space throughout the page?

Reply:
If I put space, the `split` command would interpret it differently right? It would split 2 files `nama` and `file` rather than the original doc, `split -l {{10}} filename`